src/blocks/time.rs

```diff
@@ -38,17 +38,29 @@ pub struct Config {
     format: FormatConfig,
     #[default(1.into())]
     interval: Seconds,
-    timezone: Option<Tz>,
+    timezones: Option<Vec<Tz>>,
 }
 
 pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
+    // "toggle" changes the current displayed timezone to the timezone next in the list.
+    // for example in the list ["Asia/Kolkata", "Europe/London"] if the current displayed timezone is
+    // "Europe/London" on "toggle" the timezone changes to "Asia/Kolkata"
+    api.set_default_actions(&[(MouseButton::Left, None, "toggle")]).await?;
```

Comment:
Perhaps `next_timezone` would be a better name? New actions can be added later and "toggle" is a bit too generic.

Also please document this action. For example, see [net docs](https://greshake.github.io/i3status-rust/i3status_rs/blocks/net/index.html).

Reply:
Thanks for the review! I added the documentation for `next_timezone` action.